### ISSUE TITLE
Optional internal HTTP server for get service status or other metrics

### ISSUE THREAD

**Issue by ukolovda** [cybertec-postgresql/pg_timetable#367]
**Is your feature request related to a problem? Please describe.**
I try run `pg_timetable` as service in Gitlab CI (with PostgreSQL service).
But I've got the error: 
```
Health check error:
service "runner-z52vvf7a-project-66-concurrent-1-8460e97395a3dbe4-cybertecpostgresql__pg_timetable-1-wait-for-service" health check: exit code 1
Health check container logs:
FATAL: No HOST or PORT found
```

**Describe the solution you'd like**
Make internal HTTP-service, used for checking `pg_timetable` status (health).

It also will be useful for running in Kubernetes.
And read online info (currently running tasks, total processed tasks, etc) for Prometheus diagnostics.

I think, It should be optional (by security reasons), and should not start by default.

**Describe alternatives you've considered**
I don't know.

**Additional context**

**Comment by pashagolub** [comment]
Good day,

I'm not sure I correctly understand what you're trying to achieve. Can you share what you've done so far (sources, GitLab settings, etc), please?

**Comment by ukolovda** [comment]
Thank you!
You can see https://github.com/ukolovda/pg_timetable/tree/367-check-status-by-http
I made separate branch for it, now it works fine as gitlab service.
In gitlab script I also use simple bash script for waiting status "running".

**Comment by ukolovda** [comment]
See draft PR https://github.com/cybertec-postgresql/pg_timetable/pull/368

**Comment by pashagolub** [comment]
Thanks for your draft. We decided to implement a rich REST API not only one endpoint. This issue will be addressed during work on #378
